easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.3.1093-foss-2020a-Java-11-R-4.0.0.eb

```diff
@@ -34,15 +42,32 @@ osdependencies = [
     ('pam-devel', 'libpam0g-dev')
 ]
 
+start_dir = "%(builddir)s/rstudio-%(version)s"
+separate_build_dir = False
 build_type = "Release"
-local_dep_dir = "%(builddir)s/rstudio-%(version)s/dependencies/common"
-preconfigopts = ("(cd %s && ./install-dictionaries && "
-                 "./install-pandoc && ./install-mathjax) && ") % local_dep_dir
+
+preconfigopts = " && ".join([
+    "cd %(builddir)s/rstudio-%(version)s/dependencies/common",
+    "./install-cef",
+    "./install-dictionaries",
+    "./install-mathjax",
+    "./install-pandoc",
+    "./install-packages",
+    "cd ../..",
+    "",  # To append &&
+])
+
 configopts = "-DRSTUDIO_TARGET=Server -DRSTUDIO_BOOST_SIGNALS_VERSION=2"
 
 sanity_check_paths = {
-    'files': ["bin/rstudio-server"],
+    'files': ["bin/rstudio-server", "bin/rsession", "bin/rserver", "bin/rserver-pam", "bin/rpostback"],
     'dirs': ['bin', 'extras', 'resources', 'www', 'www-symbolmaps', 'R'],
 }
 
+sanity_check_commands = [
+    "rsession --verify-installation=1",
```

Comment:
This seems to have some requirements that I don't meet so I will have to remove it:
```
[ocaisa@generoso ~]$ rsession --verify-installation=1
28 Apr 2021 09:47:32 [rsession-ocaisa] ERROR system error 2 (No such file or directory) [path: ]; OCCURRED AT rstudio::core::Error rstudio::r::session::discoverR(rstudio::r::session::RLocations*) src/cpp/r/session/RDiscovery.cpp:62; LOGGED FROM: int main(int, char* const*) src/cpp/session/SessionMain.cpp:2125
[ocaisa@generoso ~]$ R_HOME=$EBROOTR rsession --verify-installation=1
28 Apr 2021 09:47:40 [rsession-ocaisa] ERROR system error 2 (No such file or directory) [path: ]; OCCURRED AT rstudio::core::Error rstudio::r::session::discoverR(rstudio::r::session::RLocations*) src/cpp/r/session/RDiscovery.cpp:69; LOGGED FROM: int main(int, char* const*) src/cpp/session/SessionMain.cpp:2125
[ocaisa@generoso ~]$ R_HOME=$EBROOTR R_DOC_DIR=$EBROOTR/lib64/R/doc rsession --verify-installation=1
cannot find system Renviron
28 Apr 2021 09:47:52 [rsession-ocaisa] ERROR R SUICIDE: unable to open the base package
; LOGGED FROM: void {anonymous}::rSuicide(const string&) src/cpp/session/SessionMain.cpp:1148
```

Reply:
The R_HOME should be set to `$EBROOTR/lib64/R` and then the R_DOC_DIR is automatically found as well. RStudio-server ask R where it is located via `R.home`

Reply:
@ccoulombe Thanks for the hints, for me to get the test to pass I still had to explicitly set R_DOC_DIR:
```
[ocaisa@generoso ~]$ R_HOME=$EBROOTR/lib64/R rsession --verify-installation=1
01 May 2021 10:18:05 [rsession-ocaisa] ERROR system error 2 (No such file or directory) [path: ]; OCCURRED AT rstudio::core::Error rstudio::r::session::discoverR(rstudio::r::session::RLocations*) src/cpp/r/session/RDiscovery.cpp:69; LOGGED FROM: int main(int, char* const*) src/cpp/session/SessionMain.cpp:2125
[ocaisa@generoso ~]$ R_HOME=$EBROOTR/lib64/R R_DOC_DIR=$EBROOTR/lib64/R/doc rsession --verify-installation=1
[ocaisa@generoso ~]$
```